backend/controllers/createAuthUser.js

```diff
@@ -44,7 +44,9 @@ const createAuthUserFactory = (
       stashStateAndRedirectToIndex(req, res, errors, [user])
     } else {
       try {
-        const userId = await createAuthUser(res.locals, user)
+        const groupCodes = [user.groupCode]
+        const updatedUser = { ...user, groupCodes }
```

Comment:
There is a method to toArrayOrUndefined that might be useful here?

Reply:
as discussed , this is not required